packages/zoe/tools/priceAuthorityRegistry.js

```diff
@@ -0,0 +1,138 @@
+// @ts-check
+
+import { E } from '@agoric/eventual-send';
+import { makeStore } from '@agoric/store';
+import { assert, details } from '@agoric/assert';
+
+import '../exported';
+
+/**
+ * @typedef {Object} Deleter
+ * @property {() => void} delete
+ */
+
+/**
+ * @typedef {Object} PriceAuthorityRegistryAdmin
+ * @property {(pa: ERef<PriceAuthority>, brandIn: Brand, brandOut: Brand)
+ * => Deleter} registerPriceAuthority Add a unique price authority for a given
+ * pair
+ */
+
+/**
+ * @typedef {Object} PriceAuthorityRegistry A price authority that is a facade
+ * for other backing price authorities registered for a given asset and price
+ * brand
+ * @property {PriceAuthority} priceAuthority
+ * @property {PriceAuthorityRegistryAdmin} adminFacet
+ */
+
+/**
+ * @returns {PriceAuthorityRegistry}
+ */
+export const makePriceAuthorityRegistry = () => {
+  /**
+   * @typedef {Object} PriceAuthorityRecord A record indicating a registered
+   * price authority
+   * @property {ERef<PriceAuthority>} priceAuthority
+   */
```

Comment:
Hmm, why this level of indirection? Why not make the value of the store the ERef<priceAuthority> directly?